src/main/java/org/jalasoft/moi/model/csharp/CsharpCommandBuilder.java

```diff
@@ -49,4 +50,19 @@ public String buildCommand(Path path) {
         return getFolderPath(path)+" && "+COMPILER_PATH+path.getFileName().toString()+" && "+getCompiledName(path);
     }
 
+    @Override
+    public String buildCommand(List<Path> paths) {
+        return null;
+    }
```

Comment:
If the method will be null, it is better not to write it

Reply:
This is what was implemented with the interface for default